regression/contracts/quantifiers-loop-02/test.desc

```diff
@@ -1,6 +1,6 @@
-CORE
+CORE smt-backend
```

Comment:
This test works correctly either way, we explicitly provide `--z3` in the next line.

I added the tag because on systems that don't have `z3` installed and that only run SAT-specific tests, this test should be ignored. Also see #6181.

Currently test tags are not respected within `regression/contract`. I have reported this already @ https://github.com/diffblue/cbmc/issues/5986#issuecomment-845529645. When that's fixed, this test would be automatically ignored in SAT-only mode.